specifyweb/frontend/js_src/lib/components/QueryBuilder/Header.tsx

```diff
@@ -116,6 +118,9 @@ export function QueryHeader({
         </ProtectedAction>
       )}
       <div className="flex flex-wrap justify-center gap-2">
+        <Button.Small onClick={() => setIsAllCollapsed(!isAllCollapsed)}>
```

Comment:
Add `aria-pressed={isAllCollapsed}` attribute. besides accessibility, it will also provide styling